### ISSUE TITLE
Completion network input channel problem? 

### ISSUE THREAD

**Issue by BlossomingL** [otenim/GLCIC-PyTorch#11]
Hi~ Can you tell me why the completion network input channel is 4,you combine the mask and img?

**Comment by otenim** [comment]
@LinXXXX 

I also believe the last binary channel (mask) does not have to be concatenated with an RGB input image, though the paper says they do that in the proposed implementation (see section 3.2 of http://iizuka.cs.tsukuba.ac.jp/projects/completion/data/completion_sig2017.pdf), so I just followed it.

(Actually, I tried training a model with input channel = 3 just for my curiosity,
however I could not found any differences with models trained with input channel = 4.)

**Comment by zhengbowei** [comment]
Excuse me，I'd like to know if you have models running on places2 or Imagenet datasets, and if you can provide them. Thank you！
